setup.py

```diff
@@ -50,7 +50,8 @@
 		'pytest~=6.2.5',
 		'coverage~=6.2',
 		'pytest-cov~=3.0.0',
-		'coveralls~=3.3.1'
+		'coveralls~=3.3.1',
+		'tomli=2.0.0'
```

Comment:
mmh? What requires tomli?

Reply:
I am not too versed in Python packaging, but this is the message I get when trying to start `alice` from a fresh venv after doing `pip install --editable .`:

```
(venv) user@box:/usr/src/AliceCLI-orig# alice
Traceback (most recent call last):
  File "/usr/src/AliceCLI-orig/venv/bin/alice", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3252, in <module>
    def _initialize_master_working_set():
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'tomli; extra == "toml"' distribution was not found and is required by coverage

```

Reply:
Here it is:

```
# pipdeptree -p toml -r
toml==0.10.2
  **- pytest==6.2.5 [requires: toml]**
    - projectalice-cli==1.2.0 [requires: pytest~=6.2.5]
    - pytest-cov==3.0.0 [requires: pytest>=4.6]
      - projectalice-cli==1.2.0 [requires: pytest-cov~=3.0.0]
```

Reply:
That's.... weird, coverage should install it as it is its own dep. Whatever, not a big deal :-)